content/includes/installation/enable-agent-selinux.md

```diff
@@ -0,0 +1,31 @@
+---
+docs:
```

Comment:
Need to add a list of the files where this include is being used.
`files:`

see example here: https://github.com/nginx/documentation/blob/agent-release-3.0/content/includes/agent/architecture.md?plain=1